internal/controller/runner_controller.go

```diff
@@ -230,10 +230,6 @@ func (r *RunnerReconciler) EnqueueRunnerInstances(ctx context.Context, instanceC
 		return err
 	}
 
-	if len(pools.Items) < 1 {
-		return nil
-	}
-
```

Comment:
this seems to be here by mistage - as you already propose this change in https://github.com/mercedes-benz/garm-operator/pull/113/files#diff-93df91d9269bd31cc29ac5ba24b171a95bf84f12d51bc82ab4031ee47332b852L233-L236